flanker/addresslib/address.py

```diff
@@ -36,16 +36,24 @@
 
 import time
 import flanker.addresslib.parser
+import flanker.addresslib.lexer
+import ply.lex
+import ply.yacc
 from flanker.addresslib.quote import smart_unquote, smart_quote
 import flanker.addresslib.validate
+import logging
 
-from flanker.addresslib.parser import MAX_ADDRESS_LENGTH
 from flanker.utils import is_pure_ascii
 from flanker.utils import metrics_wrapper
 from flanker.mime.message.headers.encoding import encode_string
 from flanker.mime.message.headers.encodedword import mime_to_unicode
 from urlparse import urlparse
 
+log = logging.getLogger(__name__)
+
+MAX_ADDRESS_LENGTH = 1024
```

Comment:
No. It's an artificial limit I carried forward from the previous lexer. It's there to keep us from spinning to long on addresses that are abnormally long or malicious. `MAX_ADDRESS_NUMBER` and `MAX_ADDRESS_LIST_LENGTH` serve the same purpose.